test/interval_tests/numeric_tests.jl

```diff
@@ -73,9 +73,9 @@ facts("Numeric tests") do
     @fact @biginterval(-3,4) ^ 0.5 --> @biginterval(0, 2)
 
     @fact @interval(1,27)^@interval(1/3) --> roughly(Interval(1., 3.))
-    @fact @interval(1,27)^(1/3) --> Interval(1., 3.)
-    @fact @interval(1,27)^(1//3) --> Interval(1., 3.)
-    @fact @interval(0.1,0.7)^(1//3) --> Interval(0.46415888336127786, 0.8879040017426009)
+    @fact @interval(1,27)^(1/3) --> roughly(Interval(1., 3.))
+    @fact Interval(1., 3.) ⊆ @interval(1,27)^(1//3) --> true
+    @fact @interval(0.1,0.7)^(1//3) --> Interval(0.4641588833612778, 0.8879040017426008)
```

Comment:
I dislike (and worries me) the necessity to adapt the *numeric* values of the tests (e.g., line 78, and others below). While the present may well be correct, some examples should that the *new* intervals are slightly larger (or shifted!) in comparison to what we had before. 

So, are we correctly rounding, and using tests that yield the tightest interval?